src/runtime/cmd/kata-runtime/kata-exec.go

```diff
@@ -96,6 +111,35 @@ var kataExecCLICommand = cli.Command{
 	},
 }
 
+func RunCommand(cmd string, c net.Conn) (err error) {
```

Comment:
Hi @ryansavino !

I spent some time on this new feature of `kata-runtime exec` but I still don't feel confident to ack (nor nack) it. To be honest, it lacks me knowledge to judge this implementation being good or not, also I'm worried about it breaking the default behavior (just log in the VM) as well as  the `--kata-debug-port` parameter. Did you test they don't break?

Let's see if somebody in the community is confortable to ack this.

Reply:
Yes I thoroughly tested the default behavior, works fine.

The port option shouldn't be affected as this is used when the `getConn` method is called before any of this extra handling. Also, `RunCommand` only gets called if additional args are specified, so it doesn't affect the normal flow.